### ISSUE TITLE
Expand the tempo range for the metronome

### ISSUE THREAD

**Issue by Don-Stevenson** [KCarlile/flashchord#32]

**Comment by KCarlile** [comment]
@Don-Stevenson, do you mean the literal tempo range of the slider? That is, do you want it slower than 60bpm and faster than 220bpm?

Or is this intended to be a reference to the option to have accent beats on 2 and 4 like we discussed?

**Comment by Don-Stevenson** [comment]
@KCarlile  Yes, I mean to expand the tempo range in the slide.  I think about 20 to 250 BPM is probably good.

I will create an issue for the specific 2 and 4 and the approaches possible to solve that issue.

**Comment by KCarlile** [comment]
I already added the 2/4 accents to issue #31. I’m okay with going up to 250, but I think 20 is way to slow. Maybe 50, but I think Adam Neely has a video on how 33bpm is the slowest possible music. :)

**Comment by KCarlile** [comment]
I guess you haven’t seen the new feature yet were you can do multiple measures per chord. That might solve the issue of wanting to go even slower. Check out http://dev.flashchord.com

**Comment by Don-Stevenson** [comment]
@KCarlile  I saw that video of Neely's!  I really like his channel BTW.

 My rationale was thinking about using 20 as two and 4 gives 40bpm.  That said, come to think of it, when I practice slow tempos on a metronome I do think of it as being on each beat.  So maybe 33bpm, or 30 for an even number, is the low range following Neely's suggestion.

I checked out the latest dev version. The multiple measures thing is great!  

The issue with the metronome is maybe more about the direction of the project. My thought is to use this site in a way that I currently use a metronome with the added functionality of flashcards to help me practice different scales, arpeggios, key areas, etc. 

When I practice with a metronome, my goal is to make it as musically real an experience as I can muster. To do that, practising with the click on 2 and 4, on beat 3 for up-tempos, on beat 1 (or beat two) for waltzes, on 1 and 4 for 6/8, etc allows me to best approximate a musical situation while laying bare my rhythmic execution/abilities so that I can correct and improve them. I would very much like to see this functionality added to the site.

Just in case the above isn't clear. Jens Larsen gives some great examples of what I am talking about. https://www.youtube.com/watch?v=XUnu-oYWvtM

Does this clarify it all? If so, does that sound like a good direction to go in?

Sorry for the short novel BTW! :)

**Comment by KCarlile** [comment]
Adding a note from #104, as a dupe:

> Could you please extend the BPM lower range to 40 BPM (or lower) on the BPM slider?
